github-page/_guides/advanced/objectId.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Using ObjectId"
+redirect_from:
+  - /guides/advanced/objectid
+---
+
+Using the type `ObjectId` can be done by defining it as such:
+
+```ts
+import mongoose from 'mongoose';
+import { prop } from '@typegoose/typegoose';
+
+class FooModel {
+  @prop()
+  userId: mongoose.Schema.Types.ObjectId;
```

Comment:
```suggestion
  userId: mongoose.Types.ObjectId;
```

Reply:
Did we not include in #239 that it should be `mongoose.Schema.Types.ObjectId`? 🤔

Reply:
please re-read the second section of https://github.com/typegoose/typegoose/issues/239#issuecomment-618909928

Reply:
Oh, I interpreted that as only being for when querying the model. I'll update!